08-projects.Rmd

```diff
@@ -6587,18 +6587,18 @@ In this project we want to offer the flexibility of using your choice of R and/o
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
-- 1 or more SQL queries used to answer the questions.
+- SQL queries used to answer the questions.
 - How many and what are the unique `types` from the `titles` table?
-- A list of `type` and and count for the year (`premiered`) 2017.
+- A list of `type` and and count for the year (`premiered`) that had the most `titles`.
```

Comment:
Hi @kevinamstutz , I removed 2017 as the students should find the year from the previous question. However, feel free to add it back if you think it okay to include 2017 so that students who did not get the year in Question 1 can still answer Question 2.

Reply:
I agree with you